app/views/posts/show.html.erb

```diff
@@ -1,9 +1,30 @@
 <%= stylesheet_link_tag 'user', media: 'all', 'data-turbolinks-track': 'reload' %>
 <%= stylesheet_link_tag 'post', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+<body>
+<% flash.each do |key, value| %>
+  <div class="flash <%= key %>">
+    <%= value %>
+  </div>
+<% end %>
+<nav class="nav">
+    <%= link_to "Home", root_path %>
+    <%= link_to "Back", :back %>
+</nav>
+
 <%= render partial: 'post', locals: { post: @post } %> <br>
 <%= "no comments for this post" if @post.comments.empty?%>
 <div class="comments">
 <% @post.comments.each do |comment|%>
         <%=render comment %>
 <% end %>
-</div>
\ No newline at end of file
+</div>
+
+<div class="like-comment"><%= render "comments/create" %> 
+
+<%if Like.where(author_id: User.first.id, post_id: @post.id).exists?%>
+ <p id= "liked">Liked<p>
+<%else%>
+<%= render "likes/create" %></div>
+<%end%>
+</body>
```

Comment:
- [ ] Kindly make sure you have an implementation where the user can like a post, as you may notice in your current implementation there's nowhere where the user can like a post. please consider looking onto that.

![image](https://github.com/HafizMassamTabraizKhan/Blog-app/assets/97100091/13cbba12-eb16-4942-b9ba-540e85f867c3)